queries/luap/highlights.scm

```diff
@@ -1,9 +1,10 @@
-"." @character
-
 [
   (anchor_begin)
   (anchor_end)
-] @string.escape
```

Comment:
This change was not described in the PR. I'll inquire @ObserverOfTime on these kinds of decisions

Reply:
I feel like these should be `@punctuation.delimiter`. Same for their regex counterparts.

Reply:
Sounds good. Should the `.` be kept as is or also included in this capture?

Reply:
It should be kept as is.